gradlew

```diff
@@ -215,11 +215,11 @@ fi
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
 DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
 
-# Collect all arguments for the java command;
-#   * $DEFAULT_JVM_OPTS, $JAVA_OPTS, and $GRADLE_OPTS can contain fragments of
-#     shell script including quotes and variable substitutions, so put them in
-#     double quotes to make sure that they get re-expanded; and
-#   * put everything else in single quotes, so that it's not re-expanded.
+# Collect all arguments for the java command:
```

Comment:
A similar change was made in another PR and turns out that these lines may not need to be changed. Here is a reference to @ijuma's comment about it https://github.com/apache/kafka/pull/14502#discussion_r1355560355 

Can you please add what commands are you running that is leading to change in these lines?

Reply:
This is what I run to upgrade to the latest version of Gradle (as per documentation): https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper
`./gradlew wrapper --gradle-version latest`